package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "whats-that-gerber",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
Oops, merged #3 without noticing this. Can you please undo this change? I prefer to bump versions with the `npm version` command after everything is merged into master. That way Travis picks up the tag and releases everything properly